test/addons/argocd/start

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python3
+
+# SPDX-FileCopyrightText: The RamenDR authors
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+import sys
+import json
+
+from drenv import kubectl
+from drenv import minikube
+from drenv import commands
+
+
+VERSION = "release-2.11"
+URL = f"https://raw.githubusercontent.com/argoproj/argo-cd/{VERSION}/manifests/install.yaml"
+
+
+def deploy(cluster):
+    print("Deploying argocd")
+    kubectl.apply("--filename", "argocd-ns.yaml", context=cluster)
+    kubectl.apply("-n", "argocd", "--filename", URL, context=cluster)
```

Comment:
We are using now kustomization for everything, so every resource from github is cached locally. This is important for reliable builds int the e2e lab.

See how other addons are consuming resources from github and using drenv cache.

Regarding the apply command - it is not consistent with other code:
- We always use the long options (e.g. "--filename")
- We always use the same arguments order

This may not be relevant after switching to kustomization, but a general node on how to write addons.

Reply:
updated code. thanks